olexandr.vakulinskiy/HW3/Task1/prime_numbers.rb

```diff
@@ -0,0 +1,54 @@
+require 'matrix'
+class Matrix
+  def print_matrix
+    count = 0
+    each do |number|
+      if count < column_size - 1
+        print number.to_s + ', '
```

Comment:
Here and below in line 10 use [string interpolation](https://github.com/rubocop-hq/ruby-style-guide#string-interpolation).